doc/rtd/topics/bugs.rst

```diff
@@ -0,0 +1,95 @@
+.. _reporting_bugs:
+
+Reporting Bugs
+**************
+
+Logs
+====
+
+To aid in debugging collecting the necessary logs, first run the
+`collect-logs` subcommand to produce a tarfile that you can easily upload
+to aid in intial debugging of your issue:
+
+.. code-block:: shell-session
+
+  $ cloud-init collect-logs
+  Wrote /home/ubuntu/cloud-init.tar.gz
+
+If your cloud-init does not have the  `collect-logs` subcommand, then please
+manually collect the base log files by doing the following:
+
+.. code-block:: shell-session
+
+  $ dmesg > dmesg.txt
+  $ sudo journalctl -o short-precise > journal.txt
+  $ sudo tar -cvf cloud-init.tar dmesg.txt journal.txt /run/cloud-init \
+      /var/log/cloud-init.log /var/log/cloud-init-output.log
+
+Reporting
+=========
+
+Bugs for upstream cloud-init are tracked using Launchpad. To file a bug,
+`create a Launchpad account`_, login, and then
+`report an upstream cloud-init bug`_.
```

Comment:
Should we mention somewhere in here that the first thing we'll do for a bug report without debugging information is ask for the debugging information? I don't know if it's clear that we consider that a (near) necessity for looking at a bug.

Reply:
Yes, added we need logs or we will ask and not spend any further time on it.